src/components/ebay-menu/examples/16-sprites/template.marko

```diff
@@ -0,0 +1,17 @@
+<style>
+    span.flag {
+        background-image: url('https://ir.ebaystatic.com/pictures/aw/pics/cmp/ds3/sprds3_21.png');
```

Comment:
nit: I thought background images were not good for accessibility? Would this cause a problem there 
Isn't img tag a better way? The support might not be there fully, but there is object-position for img and sprites. 
I know it's just an example and not a big deal, but I think some team might end up copying this and then using it in their code.

Reply:
Yeah, I should have added this to the PR notes. The reason I am using a sprite is because that's what our global header are going to use. We don't have the flags in any other format. There is no issue with accessibility here, because the name of the country is displayed in text alongside the image. The screen reader will simply ignore the icon span. I think at some point we would get SVG versions of the flags and then create an actual country selector component. Until then this example will have to suffice. Let's monitor this example and see if it does cause confusion like you say.